modules/nw-ovn-ipsec-enable.adoc

```diff
@@ -18,6 +18,11 @@ You can configure IPsec in either of the following modes:
 If you configure IPsec in `Full` mode, you must also complete the "Configuring IPsec encryption for external traffic" procedure.
 ====
 
+The `Full` mode schema of the Cluster Network Operator (CNO) includes the `encapsulation.enum` parameter. The parameter configures the `libreswan` utility so that the utility can use User Datagram Protocol (UDP) port `4500` and network address translation-traversal (NAT-T) to encapsulate pod-to-pod traffic across nodes in your cluster. The `enum` parameter supports the following values:
```

Comment:
It can be changed like this: When IPsec in `Full` mode, then a cluster administrator can configure IPsec for `Full` mode with `full` option. 
It currently supports `encapsulation` parameter which helps to configure NAT-T encapsulation for the IPsec traffic. This parameter can have the following values:
....
....
When `full` option is omitted, then OVN uses encapsulation option set with `Auto` by default.